lib/libc/math/lib_sqrt.c

```diff
@@ -60,7 +60,9 @@
 #ifdef CONFIG_HAVE_DOUBLE
 double sqrt(double x)
 {
-	long double y, y1;
+	double y, y1;
```

Comment:
[R12] Each variable shall be declared in the new line.
[Example-Do]
```
void example(void)
{
    int i;
    int index;
```bash
[Example-Don't]
```
void example(void)
{
    int i, index;
```bash